packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -169,6 +173,17 @@ class ImageCache {
     return false;
   }
 
+  /// This method exists so that [FlutterImageCache] can check its weak cache
+  /// without changes to this class's public API.
+  @protected
+  ImageStreamCompleter _checkOtherResults(Object key) => null;
+
+  /// This method exists so that [FlutterImageCache] can add to its weak cache
+  /// once an [ImageStreamCompleter] has been created by this cache.
+  @protected
+  void _onImageStreamCompleterCreated(Object key, ImageStreamCompleter completer) {}
```

Comment:
_handleImageStreamCompleterCreated per style guide

Reply:
Removing this since we're just going to break the API